manim/mobject/types/vectorized_mobject.py

```diff
@@ -1109,11 +1138,7 @@ def get_nth_curve_function_with_length(
             sample_points = 10
```

Comment:
Technically, these two lines could be removed here given that `get_nth_curve_length_pieces` also takes care of the case where the parameter is `None`.